seqr/views/utils/variant_utils.py

```diff
@@ -133,10 +133,11 @@ def _add_locus_lists(projects, genes, add_list_detail=False, user=None):
 def _get_rna_seq_outliers(gene_ids, family_guids):
     data_by_individual_gene = defaultdict(lambda: {'outliers': {}})
 
-    outlier_data = get_json_for_rna_seq_outliers(
+    outlier_data = get_json_for_queryset(
         RnaSeqOutlier.objects.filter(
             gene_id__in=gene_ids, p_adjust__lt=RnaSeqOutlier.SIGNIFICANCE_THRESHOLD, sample__individual__family__guid__in=family_guids),
-        nested_fields=[{'fields': ('sample', 'individual', 'guid'), 'key': 'individualGuid'},]
+        nested_fields=[{'fields': ('sample', 'individual', 'guid'), 'key': 'individualGuid'}],
+        additional_values={'isSignificant': Value(True)},
```

Comment:
This is a temporary change. It avoids using the new shared function for the RNAseq response data, which changes the data to a list for each gene. We don't change the RNA outlier tag for the variants in the front end in this PR. Using the new shared function and changing the data structure will break the front end.
The risk, for now, is the shared `rnaSeqData` for the RNA volcano page in the redux state is incompatible with the state data for the variant panel. It can be solved by updating the variant panel in the next PR.

Reply:
Can you just add a comment here "TODO change to  get_json_for_rna_seq_outliers in issue #3324"